### PR TITLE
[OSDOCS-7827]: Migrate Using AWS controllers for Kubernetes (ACK) from MOBB to ROSA

### PR DESCRIPTION
**Pull request by AedinC**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-7827

Link to docs preview:
https://65028--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/rosa-mobb-using-aws-ack-tutorial

Peer review:
- [x] Peer reviewer has approved this change.

SME review:
- [ ] SME has approved this change.

### CODE REVIEW

File: rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+----
+$ read -r ACCESS_KEY_ID ACCESS_KEY < <(aws iam create-access-key \
+     --user-name $IAM_USER \
+     --query 'AccessKey.[AccessKeyId,SecretAccessKey]' --output text)
+----
+. Find the ARN of the recommended IAM policy:
++
+[NOTE]
+====
+You can find the recommended policy in each projects GitHub repo, for example https://github.com/aws-controllers-k8s/s3-controller/blob/main/config/iam/recommended-policy-arn.
```
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc:66]:
```suggestion
You can find the recommended policy in each project's GitHub repo, for example https://github.com/aws-controllers-k8s/s3-controller/blob/main/config/iam/recommended-policy-arn.
```
We can change 'repo' to 'repository' now or later (not crucial at this point).

**Review by tmalove — APPROVED** [review]

File: rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc (hunk 1 above)
Comment (tmalove) [reply rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc:66]: Remember to squash your commits before merging...thx!

**Comment by AedinC** [comment]
@michaelryanmcneill Can I please get a review of this PR whenever you get a chance? Many thanks!

File: rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+//  - Connor Wooley
+//---
+
+link:https://aws-controllers-k8s.github.io/community/docs/community/overview/[AWS Controllers for Kubernetes] (ACK) lets you define and use AWS service resources directly from Kubernetes. With ACK, you can take advantage of AWS-managed services for your Kubernetes applications without needing to define resources outside of the cluster or run services that provide supporting capabilities like databases or message queues within the cluster.
+
+ROSA clusters have a set of the ACK controllers in OperatorHub which makes it relatively easy to get started and use it. Caution should be taken as it is a tech preview product from AWS.
+
+This tutorial shows you how to use the ACK S3 controller as an example, but can be adapted for any other ACK controller that has an operator in the OperatorHub of your cluster.
+
+== Prerequisites
```
Comment (xueli181114) [inline rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc:27]: `kubectl` is also a command in below steps, it should be prerequisite too.
And no step for cluster-admin creation. Let's also make it a prerequisite.

File: rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+* AWS CLI
+* OpenShift CLI `oc`
+
+== Pre-install instructions
+
+. Set some useful environment variables:
++
+[source,terminal]
+----
+   export CLUSTER=ansible-rosa
```
Comment (xueli181114) [inline rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc:39]:
```suggestion
   export CLUSTER=<cluster name>
```
or let's highlight customer should replace the cluster name part.

File: rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+[source,terminal]
+----
+   NAME                              READY   STATUS    RESTARTS   AGE
+   ack-s3-controller-6dc4b4c-zgs2m   1/1     Running   0          145m
+----
+. If the `ack-s3-controller` is not running, restart it so that it can read the new config map and secret:
++
+[source,terminal]
+----
+$ kubectl rollout restart deployment ack-s3-controller
```
Comment (xueli181114) [inline rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc:137]:
```suggestion
$ kubectl rollout restart deployment ack-s3-controller -n ack-system
```

**Comment by xueli181114** [comment]
Tried with sts pod failed with CrashLoopBackOff
```
lixue@Xue-Lis-MacBook-Pro ~ %  kubectl -n ack-system get pods
NAME                                 READY   STATUS             RESTARTS      AGE
ack-s3-controller-6dbdbfc4b9-cdmqs   0/1     CrashLoopBackOff   5 (34s ago)   3m35s
ack-s3-controller-78b7b474d4-nbtlz   0/1     CrashLoopBackOff   5 (28s ago)   3m28s
```

**Review by michaelryanmcneill — COMMENTED** [review]
I've rewritten this document and submitted changes into #65756.

**Comment by AedinC** [comment]
Closing as work completed in #65756

**Pull request closed without merging**